ranger/core/actions.py

```diff
@@ -1225,8 +1224,13 @@ def write(string):
             write("%30s = %s\n" % (setting, getattr(self.settings, setting)))
 
         temporary_file.flush()
+        self._call_pager(temporary_file)
+
+    def _call_pager(self, file_):
         pager = os.environ.get('PAGER', ranger.DEFAULT_PAGER)
-        self.run([pager, temporary_file.name])
+        # PAGER might contain arguments, like "less -R"
+        pager = shlex.split(" ")
+        self.run(pager + [file_.name])
```

Comment:
The refactor is for fixing the argument name, adding the ability to specify multiple paths and making it public for use.

This function is so simple that one line is more than enough, but either way works.